.github/ISSUE_TEMPLATE/intent-to-ship--i2s-.md

```diff
@@ -51,6 +51,47 @@ Provide instructions for how to demo your feature such as a link to a demo page.
 Add any other information that may be relevant in determining if your feature can ship.
 -->
 
+## Build checklist
+
+This section describes considerations the implementor should keep in mind while building this technical solution.
+
+- [ ] Tests written
+- [ ] Documentation present
+<!--
+Has reference documentation been added? Have examples been added? Has an example been made available on amp.dev?
+-->
+- [ ] Cross platform testing
+<!--
+Have you tested all browsers and versions that AMP supports? https://amp.dev/support/faq/supported-browsers/
+Have you tested across mobile and desktop devices? 
+-->
+
+### Considerations checklist
+
+This section describes different cross cutting considerations that the implementor should keep in mind while building this technical solution.
+
+- [ ] Accessibility Considerations
+<!--
+Reach out to the UI and Accessibility WG for assistance. https://github.com/ampproject/wg-ui-and-a11y
+-->
+- [ ] Performance Considerations
+<!--
+Reach out to the UI and Accessibility WG for assistance. https://github.com/ampproject/wg-performance
+-->
+- [ ] Privacy Considerations
```

Comment:
For this and the following lines, can you add a comment with a call-out to the corresponding WGs/